mcs/class/corlib/corlib.dll.sources

```diff
@@ -1543,7 +1542,10 @@ corefx/SR.cs
 corefx/CompareInfo.cs
 corefx/MemoryExtensions.cs
 corefx/GlobalizationMode.cs
+corefx/Interop.GetRandomBytes.cs
```

Comment:
I think it might be more maintainable don't add this to common sources but add to the profiles where needed instead. It should be only few and for example, for monodroid we want to switch to native anyway.

(same with RandomNumberGen.ratorImplementation.Mono.cs)